calico/release-scripts/tests/test_helm_artifacts.py

```diff
@@ -18,12 +18,10 @@
 
 # Helm version is the calico/node version.
 version = versions[0]["components"]["calico/node"]["version"]
-chart_version = versions[0]["chart"]["version"]
```

Comment:
We've moved to a model where the chart doesn't receive its own version - each chart will be named after the release, and we will re-release if there are chart changes needed (or update the index to point at a new chart and archive the old for manual installation)